cli/pcluster/commands.py

```diff
@@ -58,7 +60,8 @@ def _create_bucket_with_resources(pcluster_config, json_params):
             resources = pkg_resources.resource_filename(__name__, resources_dir)
             utils.upload_resources_artifacts(s3_bucket_name, root=resources)
         if utils.is_hit_enabled_cluster(scheduler):
-            _upload_hit_resources(s3_bucket_name, pcluster_config, json_params)
+            _upload_hit_resources(s3_bucket_name, pcluster_config, storage_data.json_params)
+        _upload_dashboard_resource(s3_bucket_name, storage_data.cfn_params)
```

Comment:
let's add a new line before the `_upload_dashboard_resource`

Reply:
Done